test/upload-file-test.js

```diff
@@ -188,54 +190,106 @@ suite.addBatch({
                                     assert.ifError(err);
                                 }
                             },
-                            "and we get the file from the web interface without logging in": {
-                                topic: function(doc, feed, pair, cl) {
-                                    var cred = makeCred(cl, pair),
-                                        browser = new Browser(),
+                            "and we visit the uploads path from the web interface": {
+                                topic: function(doc) {
+                                    var callback = this.callback;
+
+                                    httputil.head("http://localhost:4815/uploads/", callback);
+                                },
+                                "it works": function(err, res) {
+                                    assert.ifError(err);
+                                },
+                                "it has a status code of 403": function(err, res) {
+                                    assert.isNumber(res.statusCode);
+                                    assert.equal(res.statusCode, 403);
+                                }
+                            },
+                            "and we try to get the non-public file with wrong extension from the web interface": {
```

Comment:
"with the wrong" :-)

Reply:
Also given the `url.split()` below shouldn't this be "with _no_ extension"?

Reply:
yes.